app/src/ui/lib/popover.tsx

```diff
@@ -44,7 +44,7 @@ interface IPopoverProps {
 
 export class Popover extends React.Component<IPopoverProps> {
   private focusTrapOptions: FocusTrapOptions
-  private containerDivRef = React.createRef<HTMLDivElement>()
+  public containerDivRef = React.createRef<HTMLDivElement>()
```

Comment:
Normally I would refactor the popover to have a `popoverContainerRef` emitter in the props since public members can get a bit hairy.. but since the long term plan is to refactor all popovers to use fixed positioning, and then that prop would not be needed, I just made it public so I could just grab of ref of the `popover` component in the parent component.